### PR TITLE
New item: wooden sign

### PR DESCRIPTION
**Pull request by ace-1331**
- added a custom sign
- added a few extra bits to increase what can be done with After/Before draw

It fits best in ItemMisc as it has similar properties than the wooden tray.
I can just revert the last commit if you prefer to have it in neck accessories.. however that makes it a lot less usable and slightly awkward with collars.

I changed the login fix to ignore this item.

I added a custom font folder and the font license that came with it, but the license can be removed. Now we can add custom fonts used with items in that folder (fonts are only loaded when first used)

Thanks to Gnarp for the assets!

### CODE REVIEW

File: BondageClub/Screens/Inventory/ItemMisc/WoodenSign/WoodenSign.js [hunk 1]
```diff
@@ -0,0 +1,105 @@
...
+}
+
+// When the text is changed
+function InventoryItemMiscWoodenSignChange() { 
+    var C = CharacterGetCurrent();
+    CharacterRefresh(C);
+    if (CurrentScreen == "ChatRoom") {
+        var Dictionary = [];
+        Dictionary.push({ Tag: "SourceCharacter", Text: Player.Name, MemberNumber: Player.MemberNumber });
+        Dictionary.push({ Tag: "DestinationCharacterName", Text: C.Name, MemberNumber: C.MemberNumber });
```
Comment (Nina-1474) [inline BondageClub/Screens/Inventory/ItemMisc/WoodenSign/WoodenSign.js:66]: Should this be TargetCharacterName?

**Review by Nina-1474 — CHANGES_REQUESTED** [review]

**Comment by ace-1331** [comment]
Indeed, fixed it.

**Review by Nina-1474 — APPROVED** [review]
I'd possibly suggest having a label/text to the left of the input boxes saying something like "First Line:" & "Second Line:". Though it's pretty obvious that's how it works after trying it once so not a big deal. Overall looks great!

**Review by Nina-1474 — CHANGES_REQUESTED** [review]
Just noticed this reinserts some lines into LayerNames.csv that had been removed under #1464.

**Comment by ace-1331** [comment]
fixed

**Review by Nina-1474 — APPROVED** [review]

**Review by elliesec — APPROVED** [review]

**Pull request merged**